### ISSUE TITLE
Allow multiple covariates

### ISSUE THREAD

**Issue by drisso** [YosefLab/scone#67]
The user should be able to adjust for other (categorical and continuous) covariates in addition to batch.

**Comment by allonw** [comment]
I assume that this means allowing any design matrix of technical covariates (maybe with some convenience functions for common cases)? If so, it should be allowed also for the bio covariates.

**Comment by drisso** [comment]
Yes. It should allow an arbitrary number of covariates to be removed and an arbitrary number of covariates to be preserved.

**Comment by drisso** [comment]
The plan is to add a single covariate for the moment. To do this:

- [ ] Add the adjust_covariate=c("no", "yes", "force") argument
- [ ] Modify parse_row() and make_design()
- [ ] Modify scone()
